library/src/main/java/com/novoda/downloadmanager/lib/GlobalState.java

```diff
@@ -20,6 +22,7 @@ public static boolean hasVerboseLogging() {
     }
 
     public static void setVerboseLogging(boolean verboseLogging) {
+        LLog.setShowLogs(verboseLogging);
```

Comment:
this is now how logging is determined to be on or off. It is off by default for builds. Can be turned on through the builder when used by clients.